tests/cephadm/test_check_non_exisiting_legacy_services.py

```diff
@@ -0,0 +1,43 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify if the cephadm shell tries to infer config from
+    a dummy mon created.
+    Verifies bz:2080242
+
+    Args:
+        ceph_cluster: Ceph cluster object
+        **kw :     Key/value pairs of configuration information to be used in the test.
+
+    Returns:
+        int - 0 when the execution is successful else 1 (for failure).
+    """
+
+    config = kw.get("config")
+
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+
+    for node in ceph_cluster.get_nodes():
+        # Create a dummy mon
+        hostname = node.hostname
```

Comment:
i think instead of creating dummy mon on all nodes, just create it on the node where mon doesn't exist

Reply:
Noted. Will do it on nodes where 'mon' isn't running